src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.cs

```diff
@@ -34,12 +37,12 @@ public void AddDependencyLocation(string fullPath)
             {
                 if (!_knownAssemblyPathsBySimpleName.TryGetValue(simpleName, out var paths))
                 {
-                    _knownAssemblyPathsBySimpleName.Add(simpleName, new List<string>() { fullPath });
-                }
-                else if (!paths.Contains(fullPath))
-                {
-                    paths.Add(fullPath);
+                    // Use case sensitive string comparer for file paths.
+                    paths = new HashSet<string>(StringComparer.Ordinal);
```

Comment:
Do we need to use `Roslyn.Utilities.PathUtilities.Comparer`?

Reply:
Done, thanks.